### ISSUE TITLE
Allow for pane scrolling

### ISSUE THREAD

**Issue by krstp** [zed-industries/community#2249]
### Check for existing issues

- [X] Completed

### Describe the feature

As a user I would like to have ability to scroll the panes tabs, such as currently when too many tabs are open within a single pane, it is not clear how many are opened. I would like to have some kind of visual indicator that allows me to scroll through those.

BTW: Currently we can scroll through tabs, but there is no clear visual indicator how many tabs are opened

![Screenshot 2023-11-08 at 4 08 50 PM](https://github.com/zed-industries/community/assets/4711498/79d342f3-2bc3-4b89-9a63-b4ed0f79f312)

### If applicable, add mockups / screenshots to help present your vision of the feature

A sample from Sublime Text...
![Screenshot 2023-11-08 at 4 06 07 PM](https://github.com/zed-industries/community/assets/4711498/fd4fb6c8-a95e-47eb-b755-048b434aba16)

There could be different ways around it.

**Comment by JosephTLyons** [comment]
Hey @krstp, what are you showing in the screenshot of Sublime Text? I'm not sure I'm seeing the indicator that shows what you are mentioning.

**Comment by krstp** [comment]
Correct, one of those is from _Sublime Text_ as per description. There is no real indicator, the only thing you can notice (see left or right of the screenshot), you will see edge tabs slightly collapsed. Contrary, in ZED, there is no clear indication there is more tabs beyond what is within the visible area. You see what I mean? If this is still not clear, I can try to make a video.

The main point to be made,: it is not clear to the user in ZED there is more tabs within given pane, visible area shows (as per above screenshot) only `5` tabs, but actually there could be `20` (God forbid) without any indication there might be more - user must be really conscious there could be more hidden upon scroll. 

I agree screenshot might not show it right away, it is more of a touch and feel.

I hope you can see my point? Thanks.
